### DISCUSSION TITLE
Axissymmetric Problem with Peridynamics Module

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#18566] [Q&A Modules: General] [answered]
Dear MOOSE Community

I wish to work with peridynamics module in order to simulate thermo-mechanical model for an axissymmetric problem regarding which I wish to know whether "FEProblem" declaration in Problems block will still work .....

Hoping to get headers in this direction.

Thanks

**Comment by hugary1995** [comment]
@hchen139

**Comment by hchen139** [comment, marked as answer]
The MOOSE peridynamics module currently does not have axisymmetric models. Please refer to the test problems of the peridynamics module on how to write an input file to use available peridynamics capabilities in MOOSE.

**Comment by ppandit95** [comment]
Ohh, Thanks for clarifying this issue.....
